database/migrations/2021_08_28_100948_create_training_place_offers_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateTrainingPlaceOffersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('training_place_offers', function (Blueprint $table) {
+            $table->uuid('offer_id');
```

Comment:
I'm not sure how I feel about this - introducing a mix of tables with uuids and tables with ids. My gut feeling is that we should stick to the convention we have in place already of ids and then use the same method as V/T to generate an offer id (see application references for info)

Reply:
I did think about it. I think my overriding motivation was not to have a public-facing auto-incrementing ID. Especially if we want to move this to nexus in the future, where we presumably might use uuids more?